Add Tim

tmwrnr opened the pull request:
## Description
Added a new extension Tim which helps to easily manage time tracking from the Raycast interface.

## Screencast
<img width="1000" alt="screenshot" src="https://user-images.githubusercontent.com/39673874/187211207-fcc6b705-e7de-432e-b52a-773452228a67.png">

https://user-images.githubusercontent.com/39673874/188087543-1b8841b3-67b9-4601-927b-89c96c4855aa.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer reviewed (changes requested):
Hi @tmwrnr 👋 

In order to be able to test this extension, it would be nice with a README explaining about Tim (and where to get it and what to do). Can you add that so It's possible to do the review?

tmwrnr commented:
Hi @pernielsentikaer,

I added a README and also added a screencast video to the initial comment. I hope now the PR is reviewable.

Let me know if you need something else.

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
